src/schema/schema_plan.c

```diff
@@ -228,6 +228,8 @@ __wt_struct_plan(WT_SESSION_IMPL *session, WT_TABLE *table, const char *columns,
                 WT_RET(__wt_buf_catfmt(session, plan, "%c", WT_PROJ_REUSE));
             current_col = col + 1;
         }
+
+        WT_RET_NOTFOUND_OK(ret);
     }
     WT_RET_TEST(ret != WT_NOTFOUND, ret);
```

Comment:
This and other instances of this in `schema_plan.c` could be changed to `WT_RET_NOTFOUND_OK`, do you think this should be done in another ticket?

Reply:
My vote is for another ticket - if the goal is to get the feature branch closed out quickly, then the fewer changes we can carry in the feature branch (and subsequently test), the better.